config/default.json

```
@@ -41,5 +41,9 @@
 	},
 	"CTL_TOOLS": {
 		"EXTERNAL_TOOL_MAX_LOGO_SIZE_IN_BYTES": 300000
+	},
+	"ADMIN_API": {
+		"ENABLED": true,
+		"ALLOWED_API_KEYS": "1ab2c3d4e5f61ab2c3d4e5f6"
```

Please change this value to some UUID, e.g.
```suggestion
		"ALLOWED_API_KEYS": "caf9afca-47ab-4ae3-9e19-01356246481c"
```